schemas/blocks/section_summary.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$id": "https://eq.ons.gov.uk/blocks/section_summary.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "block": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "https://eq.ons.gov.uk/common_definitions.json#/identifier"
+      },
+      "type": {
+        "type": "string",
+        "enum": ["SectionSummary"]
```

Comment:
The enum in `schemas/blocks/summary.json` needs to be updated (`SectionSummary` removed).

Reply:
Yep - and SectionSummary definition needed adding to the questionnaire_v1 schema. Should all be fixed in 45baaf2